pytorch_lightning/trainer/trainer.py

```diff
@@ -1016,7 +1016,7 @@ def _pre_training_routine(self):
         # print model summary
         if self.is_global_zero and self.weights_summary is not None and not self.testing:
             max_depth = ModelSummary.MODES[self.weights_summary]
-            ref_model.summarize(max_depth=max_depth)
+            summarize(ref_model, max_depth=max_depth)
```

Comment:
Here we need to put back the the log call, otherwise Lightning won't print the summary when training. 

```suggestion
            log.info("\n" + str(summarize(ref_model, max_depth=max_depth))
```

Reply:
rather fString?